versioning/BRM/extentmap.cpp

```diff
@@ -905,63 +924,46 @@ void ExtentMap::setExtentsMaxMin(const CPMaxMinMap_t& cpMap, bool firstNode, boo
         grabEMIndex(WRITE);
     }
 
-    entries = fEMShminfo->allocdSize / sizeof(struct EMEntry);
-
-    for (i = 0; i < entries; i++)
+    for (auto emIt = fExtentMapRBTree->begin(), end = fExtentMapRBTree->end(); emIt != end; ++emIt)
```

Comment:
This setExtentsMaxMin() method' loop should iterate over cpMap and use findByLBID() to reduce computation complexity.